apps/src/codebridge/FileBrowser/styles/fileBrowser.css

```diff
@@ -53,3 +68,12 @@ span.caret-container {
 .file-browser ul li span.button-bar i {
   margin-right: 4px;
 }
+
+li .button-kebab {
+  display: none;
+  height: 13px !important;
```

Comment:
why do we need !important here?

Reply:
I dunno. 🤷 

At the time, it was necessary and if I didn't have it in there it was oversizing the buttons so they were a little too tall, but presumably that was due to some other css glitch because after the extra bit of refactoring I did to address the comments, we seem to work okay w/o it. So I dropped it Whee! CSS!